src/pages/using-spark/components/autocomplete.mdx

```diff
@@ -0,0 +1,222 @@
+
+## Variants
+
+### Default
+
+<ComponentPreview
+  componentName="autocomplete--default-story"
+  hasHTML
+  minHeight="27rem"
+/>
+
+### Huge
+
+Huge Autocomplete is a larger version of the Autocomplete field.
+
+They're commonly used as a standalone feature on the page.
+
+<ComponentPreview
+  componentName="autocomplete--huge-story"
+  hasHTML
+  minHeight="27rem"
+/>
+
+## States
+
+Each Autocomplete variant has an error and disabled state.
+
+### Error
+
+<ComponentPreview componentName="autocomplete--default-invalid" hasHTML />
+
+### Disabled
+
+<ComponentPreview componentName="autocomplete--default-disabled" hasHTML />
+
+### Active Typing
+
+<img
+  class="sprk-u-mbm"
+  src={typingExample}
+  alt="As a user types into the Autocomplete input, matching results are displayed in a listbox below the input."/>
+
+### Selecting Option
+
+<img
+  class="sprk-u-mbm"
+  src={selectingExample}
+  alt="The Autocomplete component with a mouse cursor highlighting a search result."/>
+
+### No Results Found
+
+Provide guidance and offer suggestions if no results are found or do not match any information in the database.
+
+Create a better experience for the user by:
+
+- Expressing empathy.
+- Clearly communicate the state of the result.
+- Offer suggestions for next steps.
+
+<img
+  class="sprk-u-mbm"
+  src={noResultsFoundExample}
+  alt="The Autocomplete component displaying a 'No results found' message."/>
+
+<SprkDivider element="span" additionalClasses="sprk-u-Measure"></SprkDivider>
+
+## Additional Customizations
+
+### No Search Icon
+
+Showing a search icon is not required if an autocomplete is part of a larger form.
+
+Use the search icon when an autocomplete is independent of other form elements on the page to provide clarity that this is a search field.
+
+<img
+  class="sprk-u-mbm"
+  src={noSearchIconExample}
+  alt="The Autocomplete component without the search icon."/>
+
+### Address - Two Lines of Text
+
+Addresses can span two lines when displaying the full address.
+
+<img
+  class="sprk-u-mbm"
+  src={twoLinesExample}
+  alt="The Autocomplete component displaying multi-line search results."/>
+
+### No Label
+
+A label is not required for this input type. Autocomplete inputs should not use a label when used in a masthead.
+
+<img
+  class="sprk-u-mbm"
+  src={noLabelExample}
+  alt="The Autocomplete component depicted without the label."/>
+
+### Search Across Categories
+
+If there are different types of results for the same search term that users may need to look, then the categories can be featured at the top of the list for users to choose.
+
+<img
+  class="sprk-u-mbm"
+  src={categoriesExample}
+  alt="The Autocomplete component displaying category links at the top of the search results."/>
+
+### Focused - Without Search
+
+Use a focused state when users have recently interacted with the autocomplete input. By including terms they have searched in the past, we can help navigate to results quicker.
```

Comment:
Autocomplete should always be capitalized since we always capitalize components.